# Patch by Tom Harvey <TomHarvey@IndustryFigure.com>
# Rediffed. [dk]
# Also changed return type of bplay.c. [dk]

Index: errorwindow.c
===================================================================
--- errorwindow.c.orig	2012-04-20 15:46:45.454097683 +0200
+++ errorwindow.c	2012-04-20 15:47:57.307364730 +0200
@@ -11,6 +11,7 @@
 #include "buttons.h"
 #include "boxes.h"
 #include "textwindow.h"
+
 #include <string.h>
 #ifndef OLD_CURSES
 #include <ncurses.h>
Index: gramofile.c
===================================================================
--- gramofile.c.orig	2012-04-20 15:46:45.454097683 +0200
+++ gramofile.c	2012-04-20 15:46:45.491096792 +0200
@@ -64,4 +64,5 @@ main (int argc, char *argv[])
   mainmenu (startdir);
 
   finishmenu (0);
+  return 0;
 }
Index: signpr_cmf.c
===================================================================
--- signpr_cmf.c.orig	2012-04-20 15:46:45.454097683 +0200
+++ signpr_cmf.c	2012-04-20 15:46:45.491096792 +0200
@@ -89,8 +89,8 @@ cond_median_param_screen (parampointer_t
   {
     " ^: no neat interpolation.              v: broad ticks not filtered out.       ",
     " ^: less ticks detected.                v: not all of tick interpolated.       ",
-    " ^: bad following of dynamics.          v: less ticks detected.                ",
-    " ^: bad following of dynamics.          v: less ticks detected.                ",
+    " ^: bad following of dynamics.          v: fewer ticks detected.               ",
+    " ^: bad following of dynamics.          v: fewer ticks detected.               ",
     " ^: only strong ticks detected.         v: music-ticks also filtered out.      ",
     " Discard changes.                                                              ",
     " Reset default values.                                                         ",
Index: signpr_cmf2.c
===================================================================
--- signpr_cmf2.c.orig	2012-04-20 15:46:45.473097227 +0200
+++ signpr_cmf2.c	2012-04-20 15:46:45.492096768 +0200
@@ -91,9 +91,9 @@ cond_median2_param_screen (parampointer_
 
   char *helplines[8] =
   {
-    " ^: less ticks detected.                v: not all of tick interpolated.       ",
-    " ^: bad following of dynamics.          v: less ticks detected.                ",
-    " ^: bad following of dynamics.          v: less ticks detected.                ",
+    " ^: fewer ticks detected.               v: not all of tick interpolated.       ",
+    " ^: bad following of dynamics.          v: fewer ticks detected.               ",
+    " ^: bad following of dynamics.          v: fewer ticks detected.               ",
     " ^: detected tick length too short      v: detected tick length longer.        ",
     " ^: only strong ticks detected.         v: music-ticks also filtered out.      ",
     " Discard changes.                                                              ",
